src/engraving/libmscore/engravingobject.h

```diff
@@ -504,6 +505,11 @@ class EngravingObject
     {
         return isStaffText() || isSystemText() || isTripletFeel() || isPlayTechAnnotation();
     }
+
+    bool isArticulation() const
```

Comment:
It is better not to add this function. Use this comparison directly in SymbolsMetaParser

Reply:
I'm not sure I understand this. This function is the standard type-id check that all `EngravingObject`s have. The only difference is that now `Articulation` also has the sub-class `Ornament`, so this check should return true also for Ornament. Anyway, to avoid confusion, I have now returned `isArticulation()` to the way it was before (implemented via template [here](https://github.com/musescore/MuseScore/blob/master/src/engraving/libmscore/engravingobject.h#L316)) and renamed this function to `isArticulationFamily()`. This is now identical to the other cases of types/subtypes, for example `isRestFamily()` etc. Hope this is better